packages/places/ios/src/RCTAEPPlaces.m

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance with the License. You
+may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+REPRESENTATIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+*/
+
+#import "RCTAEPPlaces.h"
+#import "RCTAEPPlacesDataBridge.h"
+#import <CoreLocation/CoreLocation.h>
+@import AEPPlaces;
+
+@implementation RCTAEPPlaces
+
+static NSString *const EXTENSION_NAME = @"AEPPlaces";
+
+RCT_EXPORT_MODULE(AEPPlaces);
+
+- (dispatch_queue_t)methodQueue {
+  return dispatch_get_main_queue();
+}
```

Comment:
is this a protocol method of `RCTBridgeModule`?  i'm wondering if we should be using a background thread instead of the main thread here.  WDYT?

Reply:
Typically with react native you will add this unless the underlying package provides its own method queue, so currently all of our extensions add this bit. Here's some reading: https://reactnative.dev/docs/native-modules-ios#threading